http4s/client/src/main/scala/endpoints4s/http4s/client/Endpoints.scala

```diff
@@ -137,7 +136,13 @@ trait EndpointsWithCustomErrors
     effect.map(effect.fromEither(url.encodeUrl(urlP)))(uri =>
       entity(
         bodyP,
-        headers(headersP, Http4sRequest(method, host.withPath(host.path + uri)))
+        headers(
+          headersP,
+          Http4sRequest(
+            method,
+            host.withPath(Uri.Path.unsafeFromString(host.path.renderString + uri.renderString))
```

Comment:
Would it make sense to change our internal model so that we don’t need to go back and forth to `String` representation? (This could be addressed as a separate PR)

Reply:
Yeah I think it would! There was some discrepancies in there somewhere that resulted in test failures when I tried this earlier, but I can create a ticket after we merge!